content/academy/data_collection_basics/devtools_continued.md

```diff
@@ -8,59 +8,58 @@ paths:
 
 # [](#finish-devtools) Finish with DevTools
 
-In the first part of the DevTools tutorial, we were able to collect information about a single website from the <a href="https://www.alexa.com/topsites" target="_blank">Alexa Top Sites index</a>. If you missed the previous part, we suggest going through it to understand the basic concepts, but you don't need any of the code from there, we will start from scratch.
+In the first part of the DevTools tutorial, we were able to collect information about a single website from our <a href="https://commerce-qd83plqbj-mstephen19.vercel.app/" target="_blank">"Morgan Webstore" e-commerce site</a>. If you missed the previous part, we suggest going through it to understand the basic concepts, but you don't need any of the code from there, we will start from scratch.
 
 ## [](#looping) Looping over elements
 
 > <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Loops_and_iteration" target="_blank">Visit this tutorial</a> if you want to refresh the concept of loops in programming.
 
-First, we will use the `querySelectorAll()` function from the previous chapter to get an array of all the elements that contain the sites' data.
+First, we will use the `querySelectorAll()` function from the previous chapter to get an array of all the product elements.
 
-```js
-const sites = document.querySelectorAll('div.site-listing');
-console.log(sites);
+```JavaScript
+const products = document.querySelectorAll('a[href*="/product/"]')
+
+console.log(products);
 ```
 
-![Print all websites]({{@asset data_collection_basics/images/print-all-websites.webp}})
+![Print all product title elements]({{@asset data_collection_basics/images/print-all-product-elements.png}})
```

Comment:
Images should be linked in webp format. Dont forget to run the build script to lint + convert all images to webp.